docs/deploying-helium.md

```diff
@@ -69,7 +90,9 @@ $ az account list --output table
 You can generate an Azure Service Principal using the [`az ad sp create-for-rbac`](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create) command with `--skip-assignment` option. The `--skip-assignment` parameter limits any additional permissions from being assigned the default [`Contributor`](https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-rbac-roles) role in Azure subscription.
 
 ```bash
-$ az ad sp create-for-rbac
+$ heliumSPpw=`az ad sp create-for-rbac -n http://$heliumacrsp --query password` && echo $heliumSPpw
```

Comment:
add heilumacrsp variable to setenv.sh

Reply:
You should also run the az command to export the SP ID to an env var after the current sp create. I think there is a sample of doing that in the AKS QS

Reply:
Pushed new changes with this, let me know if anything is missing.